devel/libical/files/patch-nodocs.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 23898b6..fb7e452 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,7 @@ endif()
+ ################# build subdirs ########################
+ 
+ add_subdirectory(design-data)
+-add_subdirectory(doc)
++# add_subdirectory(doc)
```

Comment:
From KDE's extra cmake modules:

```
# ECMOptionalAddSubdirectory
# --------------------------
#
# Make subdirectories optional.
#
# ::
#
#   ecm_optional_add_subdirectory(<dir>)
#
# This behaves like add_subdirectory(), except that it does not complain if the
# directory does not exist.  Additionally, if the directory does exist, it
# creates an option to allow the user to skip it. (-DBUILD_<dir>).
```

This will be easier to propose upstream for KDE projects, but in case of libical it adds a new dependency. In all cases it will depend a lot on upstream goodwill, esp. with KDE where "doc" subdirectories usually build the project's "handbook" which is available through a menu (and that menu will be there regardless of whether we build the handbook).